config/deploy/addons.managed.openshift.io_addons.yaml

```diff
@@ -61,7 +61,7 @@ spec:
                     - catalogSourceImage
                     - namespace
                     type: object
-                  OLMOwnNamespace:
+                  olmOwnNamespace:
                     description: OwnNamespace config parameters. Present only if Type = OLMOwnNamespace.
```

Comment:
Should we also update the description here?

```suggestion
                    description: OwnNamespace config parameters. Present only if Type = olmOwnNamespace.
```

Reply:
Type stays uppercase, just the config key is lowercased as all k8s json/yaml keys are.
I just also added a change to prefix the doc-strings with `OLM-` as well to be consistent.

IMHO we are referring to the constant/type key value `OLMOwnNamespace` so I would leave it like that.

To check with k8s upstream:
> Rolling update config params. Present only if DeploymentStrategyType = RollingUpdate.
> https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#deploymentstrategy-v1-apps

Also an option, but I rather have something `grep`-able, WDYT?